tests/robustness/attackers/test_classification_attackers.py

```diff
@@ -0,0 +1,103 @@
+
+import warnings
+warnings.filterwarnings("ignore")
+
+import pandas as pd
+
+from holisticai.datasets import load_dataset
+from sklearn.preprocessing import StandardScaler
+from sklearn.linear_model import LogisticRegression
+from holisticai.robustness.attackers import HopSkipJump, ZooAttack
+from holisticai.utils import BinaryClassificationProxy
+from holisticai.robustness.attackers.classification.commons import format_function_predict_proba
+from holisticai.robustness.metrics import adversarial_accuracy, empirical_robustness
+from sklearn.metrics import accuracy_score
+import pytest
+
+SHARD_SIZE=60
+
+@pytest.fixture
+def categorical_dataset():
+    dataset = load_dataset("adult", protected_attribute='race') # x,y,group_a,group_b
+    dataset = dataset.groupby(['y','group_a']).sample(SHARD_SIZE, random_state=42) # 0-ga | 0-gb  | 1-ga | 1-gb
+    train_test = dataset.train_test_split(test_size=0.02, stratify=dataset['y'], random_state=0)
+    train = train_test['train']
+    test = train_test['test']
+    correlations = train['X'].corrwith(train['y']).sort_values(ascending=False)
+    top_10_features = correlations.head(10).index.tolist()
+    train['X'] = train['X'][top_10_features]
+    test['X'] = test['X'][top_10_features]
+    return train['X'], test['X'], train['y'], test['y']
+
+
+def test_hsj(categorical_dataset):
+    train_X, test_X, train_y, test_y = categorical_dataset
+
+    # Standardize data and fit model
+    scaler = StandardScaler()
+    X_train = scaler.fit_transform(train_X)
```

Comment:
X_train = scaler.fit_transform(train["X"]) the same for test

Reply:
updated with pipeline